1config-ui/src/com/brunobonacci/oneconfig/ui/view.cljs

```diff
@@ -47,7 +47,7 @@
   (for [item items]
     (let [{:keys [key env version change-num content-type]} item]
       ^{:key (string/join "-" [key env version change-num content-type])}
```

Comment:
maybe `content-type` shouldn't be part of the key here `{:key (string/join "-" [key env version change-num content-type])}`

Reply:
yeah, we can exclude it